libnavui-maps/build.gradle

```
@@ -39,6 +39,7 @@ android {
 
 dependencies {
     api project(":libnavui-base")
+    implementation project(":libnavui-shield")
```

```suggestion
    api project(":libnavui-shield")
```
Let's expose this dependency. Developers will need it to call
```
MapboxRoadNameView#renderRoadNameWith(expectedShields: List<Expected<RouteShieldError, RouteShieldResult>>)
```
which is part of the public API of this module.